web3.js/test/websocket.test.ts

```diff
@@ -79,5 +79,24 @@ if (process.env.TEST_LIVE) {
 
       await connection.removeSignatureListener(id);
     });
+
+    it('reconnects once subscription are created', async () => {
+      const newConnection = new Connection(url);
+
+      // wait for websocket to disconnect
```

Comment:
websocket will start disconnected and will only connect when a subscription is added